test/channel/base_test.rb

```diff
@@ -92,6 +111,23 @@ def rm_rf
     assert_nil @channel.last_action
   end
 
+  test "try calling a public method defined on Base" do
+    @channel.perform_action 'action' => :kick
+    assert_nil @channel.last_action
```

Comment:
I still think it's tough to grasp that the action is never reached. Maybe extending the test name a little would do the trick?